src/System.Security.Cryptography.Algorithms/tests/DefaultRSAProvider.cs

```diff
@@ -43,7 +43,7 @@ public bool SupportsSha2Oaep
             get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows); }
         }
 
-        public bool SupportsKeyGeneration => !RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
+        public bool SupportsKeyGeneration => true;
```

Comment:
Apple doesn't support DSA.  But RSA and ECDSA are full on again now, so I'll go ahead and remove the conditionals.